tls/s2n_client_extensions.c

```diff
@@ -26,9 +26,9 @@
 
 #include "utils/s2n_safety.h"
 #include "utils/s2n_blob.h"
+#include "tls/s2n_client_extensions.h"
```

Comment:
Yes, it is used by `s2n_client_cert_req_recv` in https://github.com/awslabs/s2n/pull/425